pypred/parser.py

```diff
@@ -12,6 +12,8 @@
     'and': 'AND',
     'or': 'OR',
     'not': 'NOT',
+    'anyof': 'ANY',
```

Comment:
Have you found any supported operators in pypred that are in 2 words? I'm having a hard time understanding how `contains` and `anyof`/`allof` can be registered as distinct operators, and have all of this working. Doesn't the AST become something like : 
`left contains anyof right`

```
           contains
left -                   - anyof
                                             - right
```

Reply:
There is `is not` that's in two words, and that's the only one I think. 

This is for registering a reserved word in the parser so that it can be mapped as a token and not a `STRING` token (see `t_STRING(t)` function in parser.py) [Link](https://github.com/dialoguemd/pypred/pull/2/files#diff-8a01c43e06ad9126fa1060076a598a9abde2ec91bbbed0a75a7ee1fff0882295R84).

The magic happens in the functions starting with `p_`. In the `p_contains_any(p)` function, the documentation line (`term : factor CONTAINS ANY factor` ) tells the parser that `factor CONTAINS ANY factor` should be mapped to a term and when meeting that term, execute the body of the function. This is the way `ply` is working, so I don't see another to do it without deep diving in `ply` implementation.

Reply:
As discussed at standup, let's address this in a separate story.